types/module/module.go

```diff
@@ -454,6 +457,9 @@ func (m Manager) RunMigrations(ctx sdk.Context, cfg Configurator, fromVM Version
 				return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidType, "expected %T, got %T", configurator{}, cfg)
 			}
 
+			ctx.Logger().Info(fmt.Sprintf("Running migration for module: %s, InitGenesis (on default genesis) toVersion %d",
+				moduleName, toVersion))
```

Comment:
Why not using key vals ?

Reply:
```suggestion
			ctx.Logger().Info(fmt.Sprintf("Adding new module and initializing default genesis",
				"module", moduleName, "target_version", toVersion))
```

Reply:
I think running migration makes more sense here. In most cases its a migration and not an add. Maybe we can say Running/Adding but it would be confusing.